src/main/kotlin/io/github/georgberky/maven/plugins/depsupdate/VersionUpdate.kt

```diff
@@ -13,7 +13,7 @@ abstract class VersionUpdate(
     fun canSkip() : Boolean {
         println("canSkip: ${latestVersion} compare to ${version}")
         println("latestVersion.equals(version) = ${latestVersion.equals(version)}")
-        return !latestVersion.equals(version)
+        return latestVersion != "null" && latestVersion != version
```

Comment:
Comparing to `"null"` here seems strange to me. It looks like `null` turned to a String and I'd rather catch this earlier than having to check for a special value there. For example we could turn `lastestVersion` into a `String?`.

Reply:
This will clash with your uber-PR in progress, so I'd like to  keep this as minimal as possible for less merge clutter. Yes, it could be skipped earlier – but where? I would not want to rewrite that much code for a simple NPE fix.